src/TritonRoute/src/TritonRoute.tcl

```diff
@@ -34,17 +34,120 @@
 #############################################################################
 
 sta::define_cmd_args "detailed_route" {
-    -param filename
+    -guide filename
+    [-outputguide filename]
+    [-outputmaze filename]
+    [-outputDRC filename]
+    [-outputCMap filename]
+    [-dbProcessNode name]
+    [-drouteEndIterNum iter]
+    [-drouteViaInPinBottomLayerNum num]
+    [-drouteViaInPinTopLayerNum num]
+    [-OR_SEED seed]
+    [-OR_K k]
+    [-bottomRoutingLayer layer]
+    [-topRoutingLayer layer]
+    [-initRouteShapeCost cost]
+    [-verbose level]
+    [-param filename]
```

Comment:
`OR_SEED` and `OR_K` have a comment saying "to be removed". Not sure if I should keep them or remove them.

Reply:
Or what to do with the rest of these rarely-used options.

Reply:
BX said that he wanted to remove them but ABK wanted to keep them for future parallel exploration.  My feeling is to keep those two as undocumented arguments for now.

I would ignore the depracated ones and  drop initRouteShapeCost.  I converted top and bottom routing layers to use names rathe than numbers.  Please convert any remaining ones similarly.

Reply:
In general we use snake case for options so lets be consistent with these argument names.

Reply:
Sounds good. I just copied the names directly from the param file syntax, but I will update.

Reply:
I removed `initRouteShapeCost` but kept everything else for now. We can always remove them later.